### ISSUE TITLE
Matrix solve

### ISSUE THREAD

**Issue by grahamlopez** [project-asgard/asgard#17]
ASGarD
    asgardasgard
    Issues
    #17

Open
Opened 5 months ago by McDaniel, Benjamin T
Close issue
New issue
Matrix solve

Add a routine in the matrix class that for a matrix A and a vector argument x returns a vector b = Ax.
Related issues
0

    Graham Lopez @bgl added someday label 5 months ago
    Graham Lopez @bgl added help_wanted label 3 months ago

*given arguments "A" and "b" return "x" for Ax = b.

**Comment by bmcdanie** [comment]
we have this in fast math @grahamlopez - do we want to add tensor class method as well?

**Comment by grahamlopez** [comment]
no, not until we have a reason to need it.
